lib/Models/Terria.ts

```diff
@@ -404,15 +404,19 @@ export default class Terria {
         runInAction(() => {
           if (config.parameters) {
             this.updateParameters(config.parameters);
+            Internationalization.initLanguage(
+              config.parameters.languageConfiguration,
+              options.i18nOptions
+            );
           }
 
-          Internationalization.initLanguage(
-            config.parameters.languageConfiguration,
-            options.i18nOptions
-          );
-
           if (config.aspects) {
```

Comment:
Should `config.aspects` be tested before `config.parameters`?

Reply:
You're right in that there isn't a logical issue - if it's a magda config, `config.parameters` will never exist - however I've clarified that in https://github.com/TerriaJS/terriajs/pull/4657/commits/f5399c8f3c69f8d2903f944eca300dd0cd7d58b3 now